src/sql/command.py

```diff
@@ -55,6 +56,21 @@ def __init__(self, magic, user_ns, line, cell) -> None:
         else:
             add_alias = False
 
+        if not (add_conn or add_alias) and one_arg:
+            first_arg = self.args.line[0]
+            # Getting variable value when `%sql {{variable}}` is passed
+            if first_arg.startswith("{{") and first_arg.endswith("}}"):
+                first_arg = user_ns[first_arg[2:-2]]
```

Comment:
unclear about this part. is this some new behavior? or is it refactoring of existing one?

Reply:
Finding the variable from user_ns exists in the later part of the code with `self._var_expand`: [here](https://github.com/ploomber/jupysql/blob/master/src/sql/command.py#L70)

So I relocated the code for checking nonidentifier after `_var_expand`. Now, the condition for checking `{{` and `}}` and `[` and `]` (for the deprecated feature) are gone. : [fixed](https://github.com/ploomber/jupysql/pull/821/files#diff-769097f5e36be57ddec10c78df0900db8c1e254f5485bc94367aa7fcb07991c5R85-R91)